TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/Drivetrain.java

```diff
@@ -1,26 +1,32 @@
 package org.firstinspires.ftc.teamcode.subsystems;
 
 import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.IMU;
 
 import org.firstinspires.ftc.teamcode.hardware.RobotHardware;
 
 public class Drivetrain{
-    private final DcMotor motorFrontRight;
-    private final DcMotor motorFrontLeft;
-    private final DcMotor motorBackRight;
-    private final DcMotor motorBackLeft;
-
+    private final DcMotor motorRight;
+    private final DcMotor motorLeft;
+    private final IMU imu;
     public Drivetrain(){
-        motorFrontRight = RobotHardware.motorFrontRight;
-        motorFrontLeft= RobotHardware.motorfrontLeft;
-        motorBackLeft= RobotHardware.motorbackLeft;
-        motorBackRight = RobotHardware.motorbackRight;
+        imu = RobotHardware.DTimu;
+        motorRight = RobotHardware.DTmotorRight;
+        motorLeft= RobotHardware.DTmotorLeft;
+    }
+
+    public static void arcadeDrive(double xSpeed, double zRotation){
+        xSpeed = Math.max(-1.0, Math.min(1.0, xSpeed));
+        zRotation = Math.max(-1.0, Math.min(1.0, zRotation));
+
+        double leftSpeed = xSpeed - zRotation;
+        double rightSpeed = xSpeed + zRotation;
+
+        RobotHardware.setDrivetrainPower(leftSpeed, rightSpeed);
```

Comment:
it's not a good approach to set a value to a param. create a new variable instead